flag/service/kubeconfig/kubeconfig.go

```diff
@@ -0,0 +1,6 @@
+package kubeconfig
+
+// KubeConfig is a data structure to hold kubeconfig specific configuration flags.
+type KubeConfig struct {
+	ResourceNamespace string
```

Comment:
Let's structure the flags more so it's `config.Flag.Service.KubeConfig.Resource.Namespace`. Structuring the config makes it easier to manage.

See https://github.com/giantswarm/app-operator/blob/6b9e21d1133b4a859c7b9c5a3b18cb0a6dcb311a/flag/service/appcatalog/index/index.go for an example.